service/controller/machinepool/handler/migration/testdata/old_machinepool.yaml

```diff
@@ -0,0 +1,93 @@
+apiVersion: exp.cluster.x-k8s.io/v1alpha3
```

Comment:
Test input, old experimental MachinePool from which the migration handler will copy the CR Status.